functional_requirements/requirements.txt

```diff
@@ -1,111 +1,188 @@
 
-The watch shall display the current time using clock with hands display (hours, minutes and seconds hands)
-The watch shall change to "Clock with digits" when button A is pressed.
-The watch shall change to "Change display view" view when button B is pressed.
 
-- Clock with digits: 
+
+PART 2 Display views
+
+REQ 2.1 views
+
+The views provided by the watch shall be the following ones:
+
+- Clock view
+- Display Selection view
+- Step Count view
+- Hearth rate view
+- ECG view
+- Sp02 view
+- Map view
+- Setting view
+
+REQ 2.2 Initial view
+
+The initial display state upon power-on shall be the clock view.
+
+REQ 2.3 Clock view
 
 The watch shall display the current time using clock with digits (hours and minutes)
 The watch shall display the battery life indicator
-The watch shall change to "Clock with hands" when button A is pressed.
-The watch shall change to "Change display view" view when button B is pressed.
-
+The watch shall change to "Display Selection view" view when button B is pressed.
 
-- Change display view:
+REQ 2.4 Display Selection view
 
 Display options are: 
-	News
-	Weather
+
+	Clock
+	Step count
 	Heart rate
 	ECG
 	Sp02
 	Map
 	Settings
 	Power off
 	
+Default option is "Clock"	
 The watch shall change selected view when button A is pressed.
 The watch shall activate the selected view when button B is pressed. 
 The watch shall deactivate when "Power off" is selected.
 
+REQ 2.5 Step Count view
 
-- News view:
-
-The watch shall connect to the mobile phone to retrieve the news info through the bluetooth connection. If connection is available, the news page is displayed otherwise an error message is displayed.
-The watch shall change to "Change display view" view when button B is pressed.
-
-- Weather view:
-
-The watch shall connect to the mobile phone to retrieve the weather info through the bluetooth connection. If connection is available, the weather page is displayed otherwise an error message is displayed.
+The watch shall display the current step count. If not available, an error message is displayed.
 The watch shall change to "Change display view" view when button B is pressed.
 
-- Heart rate view:
+REQ 2.6 Heart rate view
 
 The watch shall display the current heart rate. If not available, an error message is displayed.
 The watch shall change to "Change display view" view when button B is pressed.
 
-- ECG view:
+REQ 2.7 ECG view:
 
 The watch shall display the ECG. If not available, an error message is displayed.
 The watch shall change to "Change display view" view when button B is pressed.	
-	
 
-- Map view:
+REQ 2.8 Sp02 view:
 
-The watch shall connect to the mobile phone to retrieve the map info through the bluetooth, according to the position indicated by the GPS tracker. If connection is not available or GPS position is not available with enough precision (100 m), an error message is displayed.
+The watch shall display the Sp02. If not available, an error message is displayed.

```

Comment:
Might be worth mentioning that Sp02 is blood oxygen level (%) ... or perhaps we can just put it in the text of the report

https://help.fitbit.com/articles/en_US/Help_article/2459.htm

Reply:
Fixed